build_tools/github_actions/ci_build_bazel.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Maybe I should remove this?

Reply:
Your call. It doesn't look like this file is derived from any script in tensorflow/tensorflow, but it also doesn't hurt to keep the boilerplate.